app/scripts/controllers/network/network-controller.js

```diff
@@ -189,23 +181,19 @@ export default class NetworkController extends EventEmitter {
       return;
     }
 
-    // NOTE: This will never happen in practice because you can't pass null or
```

Comment:
Now that `getCurrentChainId` doesn't exist and we are pulling `chainId` directly from the store, `chainId` can be undefined.

Reply:
This comment was _about_ the value in the store though. It is initialized with a valid chain ID, and all built-in networks have a valid chain ID, and we have UI validation to ensure you can't pass in `null`/`undefined`. So it should still hold true, barring some bug in our add network flow I guess.

Reply:
Hmm, I suppose so. I believe when I added this comment I thought it was impossible for `chainId` to be `null` or `undefined`. In hindsight, perhaps I was wrong. In any case, it definitely is possible now; you just need to initialize the controller with a state object that doesn't have a `chainId` property, which was mistakenly happening in some tests (hence the changes below to add fake `chainId`s).